docs/src/pages/[platform]/connected-components/liveness/troubleshooting/index.page.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Face Liveness Troubleshooting
+description: Troubleshoot your issues with the Amplify UI FaceLivenessDetector component.
+supportedFrameworks: react|android|swift
+---
+
+import { Fragment } from '@/components/Fragment';
+
+## Troubleshooting / Frequently Asked Questions
+
+The following issues are presented with corresponding troubleshooting steps. Please check them out in case you run into issues like these -- if the remedies don't work, please file an issue in our GitHub repositories.
+
+### Error: Cognito user pool does not allow access to unauthenticated identities
+
+First, make sure you have followed the [prerequisites for setting up Amplify resources](#step-1-configure-auth). Second, you can also go into the Cognito identity pool configured in your Amplify app, edit it and select the checkbox labeled "Enable access to unauthenticated identities" -- then save the changes. Finally, ensure that the configuration file -- `aws-exports.js` on web or `amplifyconfiguration.json` on mobile -- contains the name of the correct identity pool.
```

Comment:
I think we have to update this link now that it's on a different page?

Reply:
good catch, I think this has to be `./#step-1-configure-auth` to match the others -- unless there's an issue with routing causing the 404 in which case probably `../#step-1-configure-auth`